src/CR.cc

```diff
@@ -73,7 +73,7 @@ class IdSpan {
 /**
  * comparison functor; sort by ID, then by row (y), then by column range start (x0)
  */
-struct IdSpanCompar : public std::binary_function<const IdSpan::ConstPtr, const IdSpan::ConstPtr, bool> {
+  struct IdSpanCompar {
```

Comment:
Is it really as simple as just removing the `binary_function` bit here?  Huh!

Reply:
Looks like copy/paste from existing code. Unless I am missing something.